dagger-core/src/main/java/io/odpf/dagger/core/stream/StreamConfig.java

```diff
@@ -0,0 +1,128 @@
+package io.odpf.dagger.core.stream;
+
+import org.apache.flink.connector.kafka.source.enumerator.initializer.OffsetsInitializer;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.stream.JsonReader;
+import io.odpf.dagger.common.configuration.Configuration;
+import lombok.Getter;
+import org.apache.kafka.clients.consumer.OffsetResetStrategy;
+
+import java.io.StringReader;
+import java.util.Map;
+import java.util.Properties;
+import java.util.regex.Pattern;
+
+import static io.odpf.dagger.common.core.Constants.INPUT_STREAMS;
+import static io.odpf.dagger.core.utils.Constants.SOURCE_KAFKA_CONSUME_LARGE_MESSAGE_ENABLE_DEFAULT;
+import static io.odpf.dagger.core.utils.Constants.SOURCE_KAFKA_CONSUME_LARGE_MESSAGE_ENABLE_KEY;
+import static io.odpf.dagger.core.utils.Constants.SOURCE_KAFKA_MAX_PARTITION_FETCH_BYTES_DEFAULT;
+import static io.odpf.dagger.core.utils.Constants.SOURCE_KAFKA_MAX_PARTITION_FETCH_BYTES_KEY;
+
+public class StreamConfig {
+    private static final Gson GSON = new GsonBuilder()
+            .enableComplexMapKeySerialization()
+            .setPrettyPrinting()
+            .create();
+
+    private static final String KAFKA_PREFIX = "source_kafka_consumer_config_";
+
+    @SerializedName("SOURCE_KAFKA_TOPIC_NAMES")
+    @Getter
+    private String kafkaTopic;
+
+    @SerializedName("INPUT_SCHEMA_PROTO_CLASS")
+    @Getter
+    private String protoClass;
+
+    @SerializedName("INPUT_SCHEMA_TABLE")
+    @Getter
+    private String schemaTable;
+
+    @SerializedName("SOURCE_KAFKA_CONSUMER_CONFIG_AUTO_COMMIT_ENABLE")
+    @Getter
+    private String autoCommitEnabled;
+
+    @SerializedName("SOURCE_KAFKA_CONSUMER_CONFIG_AUTO_OFFSET_RESET")
+    @Getter
+    private String autoOffsetReset;
+
+    @SerializedName("SOURCE_KAFKA_CONSUMER_CONFIG_GROUP_ID")
+    @Getter
+    private String consumerGroup;
+
+    @SerializedName("SOURCE_KAFKA_CONSUMER_CONFIG_BOOTSTRAP_SERVERS")
+    @Getter
+    private String sourceBootstrapServers;
```

Comment:
can we keep the consistency ? in some variables we are adding source, in some we are not, we can keep them same as config names as well

Reply:
I guess some keys are inconsistent beforehand. Not able to remember the reason rn. But I have added the `Source` prefix in all newly added streams constant.